sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpConstants.java

```diff
@@ -19,4 +19,8 @@
 
     static final int AMQP_BATCH_MESSAGE_FORMAT = 0x80013700; // 2147563264L;
 
+    public static final String REQUEST_RESPONSE_LEGACY_STATUS_CODE = "status-code";
```

Comment:
This already exists in RequestResponseChannel.java. Is there a reason it is here?